components/App.js

```diff
@@ -149,7 +150,8 @@ class App extends React.Component {
               </Typist>
             </div>
 
-            <MapChart setMarker={this.setMarker} />
+            <MapChart marker={this.state.marker} setMarker={this.setMarker} />
+            <DetailView marker={this.state.marker} setMarker={this.setMarker} />
```

Comment:
`App` is now the only component managing marker state

Reply:
👍